plugins/woocommerce-blocks/assets/js/blocks/product-collection/utils.tsx

```diff
@@ -18,13 +18,14 @@ import {
  * Internal dependencies
  */
 import {
```

Comment:
Why are the `type` prefixes removed?

```suggestion
import type {
```

Reply:
I removed it because `ProductCollectionUIStatesInEditor` is an enum. Therefore, if I add the type prefix, it shows a TS error wherever it is used.

```
'ProductCollectionUIStatesInEditor' cannot be used as a value because it was imported using 'import type'.ts(1361)
```

Therefore, I had to import value instead of type. Does it make sense? Is there alternative solution? Also, Is there benefit of using type prefix in import statement?